book.rb

```diff
@@ -0,0 +1,13 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(_rental)
+    Rental.new(date, self, person)
+  end
```

Comment:
- [ ] You used date and person without defining it, I will encourage you to add it as a parameter in the method so it would be more like this `def add_rental(person, date)`, the self is a reserved word in ruby but others should be defined.